strr-pm-web/app/components/form/platform/Details.vue

```diff
@@ -0,0 +1,110 @@
+const { platformDetails, platformDetailsSchema } = storeToRefs(useStrrPlatformDetails())
+
+const radioOptions = [
+  { value: ListingSize.THOUSAND_OR_MORE, label: tPlat('text.thousandOrMore') },
+  { value: ListingSize.UNDER_THOUSAND, label: tPlat('text.lessThanThousand') }]
+
+const platformBrandRef = ref()
+watch(platformBrandRef, (val) => {
+  if (val && isComplete) {
+    val.validate()
+  }
+})
+</script>
+
+<template>
+  <div data-testid="platform-details">
+    <ConnectPageSection
+      class="bg-white"
+      :heading="{ label: tPlat('section.title.details'), labelClass: 'ml-6 font-bold' }"
+    >
+      <UForm
+        ref="platformBrandRef"
+        :state="platformDetails"
+        :schema="platformDetailsSchema"
+        class="space-y-10 pb-10"
+      >
+        <div
+          v-for="brand, i in platformDetails.brands"
+          :key="'brand' + i"
+        >
+          <ConnectSection :title="tPlat('section.subTitle.brand') + ((i > 0) ? ` ${ i + 1 }` : '')">
+            <div class="space-y-5">
+              <div class="flex flex-col gap-5 sm:flex-row-reverse">
+                <div>
+                  <UButton
+                    v-if="i !== 0"
+                    :label="t('word.Remove')"
+                    class="border border-blue-500 sm:border-0"
+                    color="primary"
+                    trailing-icon="i-mdi-close"
+                    variant="ghost"
+                    @click="removeBrandAtIndex(i)"
+                  />
+                </div>
+                <div class="grow space-y-5">
+                  <UFormGroup
+                    :aria-label="(i > 0) ? tPlat('label.brandNameOpt') : tPlat('label.brandName')"
+                    :name="`brands.${i}.brandName`"
+                    :error="isComplete && !brand.name ? 'Required' : ''"
+                  >
+                    <ConnectField
+                      :id="'platform-brand-name-' + i"
+                      v-model="brand.name"
+                      :help="tPlat('hint.brandName')"
+                      :name="`brands.element.test`"
+                      :placeholder="(i > 0) ? tPlat('label.brandNameOpt') : tPlat('label.brandName')"
+                      :wrap-with-form-grp="false"
+                    />
+                  </UFormGroup>
+                  <UFormGroup
+                    :aria-label="(i > 0) ? tPlat('label.brandSiteOpt') : tPlat('label.brandSite')"
+                    :name="`brands.${i}.website`"
+                    :error="isComplete && !brand.website ? 'Required' : ''"
+                  >
+                    <ConnectField
+                      :id="'platform-brand-site-' + i"
+                      v-model="brand.website"
+                      :help="tPlat('hint.brandSite')"
+                      name="website"
+                      :placeholder="(i > 0) ? tPlat('label.brandSiteOpt') : tPlat('label.brandSite')"
+                      :wrap-with-form-grp="false"
```

Comment:
will the help text still be rendered when wrap-with-form-grp=false?

Reply:
reworked this a bit